src/data/polynomial/mirror.lean

```diff
@@ -118,9 +118,16 @@ lemma mirror_mirror : p.mirror.mirror = p :=
 polynomial.ext (λ n, by rw [coeff_mirror, coeff_mirror,
   mirror_nat_degree, mirror_nat_trailing_degree, rev_at_invol])
 
+variables {p q}
+
+lemma mirror_eq_iff : p.mirror = q ↔ p = q.mirror :=
```

Comment:
It might make sense to add a `function.involutive mirror` statement, for which I assume this falls out as one of the lemmas.

Reply:
Thanks!